media/static-bundles.json

```diff
@@ -1046,10 +1045,9 @@
     },
     {
       "files": [
-        "protocol/js/protocol-modal.js",
         "js/libs/spin.min.js",
         "js/base/send-to-device.js",
-        "js/exp/mobile.js"
+        "js/firefox/mobile/mobile.js"
```

Comment:
Should this be `media/js/exp/mobile.js` still? That file still exists and has been updated in this PR?